lib/dom_parser.rb

```diff
@@ -0,0 +1,164 @@
+#require 'pry-byebug'
+require_relative "./dom_tree.rb"
+require_relative "./html_reader.rb"
+require_relative "./html_writer.rb"
+require_relative "./node_renderer.rb"
+require_relative "./node_searcher.rb"
+require_relative "./stack.rb"
```

Comment:
Looks like you changed this file name - should be "./Stack.rb" here and everywhere else.